pandas/tests/test_expressions.py

```diff
@@ -380,3 +380,33 @@ def test_frame_series_axis(self, axis, arith):
 
         result = op_func(other, axis=axis)
         tm.assert_frame_equal(expected, result)
+
+    @pytest.mark.parametrize(
+        "op",
+        [
+            "__mod__",
+            pytest.param("__rmod__", marks=pytest.mark.xfail(reason="GH-36552")),
+            "__floordiv__",
+            "__rfloordiv__",
+        ],
+    )
+    @pytest.mark.parametrize(
+        "box, tester",
+        [
+            (DataFrame, tm.assert_frame_equal),
+            (Series, tm.assert_series_equal),
+            (Index, tm.assert_index_equal),
+        ],
+    )
+    @pytest.mark.parametrize("scalar", [-5, 5])
+    def test_python_semantics_with_numexpr_installed(self, op, box, tester, scalar):
+        # https://github.com/pandas-dev/pandas/issues/36047
+        expr._MIN_ELEMENTS = 0
```

Comment:
I see it is done like that in other tests as well (so doesn't need to be fixed here), but that doesn't seem a very clean way to patch this, as it will influence other tests as well

Reply:
this is an class with a teardown_method that resets _MIN_ELEMENTS

Reply:
but yes we should replace with a fixture.

Reply:
Ah, didn't see the teardown/setup, only the import at the top ..